docs/extensibility/wizard.md

```diff
@@ -0,0 +1,74 @@
+# Config Map for wizard extensions
+
+**Table of Contents**
+
+- [Overview](#overview)
+- [Extension version](#extension-version)
+- [_general_ section](#general-section)
+- [_steps_ section](#steps-section)
+- [_defaults_ section](#defaults-section)
+
+## Overview
+
+This document describes the required ConfigMap setup that you need to configure in order to handle a custom wizard.
+You can provide all the ConfigMap data sections as either JSON or YAML.
+
+## Extension version
+
+The version is a string value that defines in which version the extension is configured. It is stored as a value of the `busola.io/extension-version` label. If the configuration is created with the **Create Extension** button, this value is provided automatically. When created manually, use the latest version number, for example, `'0.5'`.
+
+> **NOTE:**: Busola supports only the two latest versions of the configuration. Whenever a new version of the configuration is proposed, go to your Extension and migrate your configuration to the latest version.
+
+## _general_ section
+
+The **general** section is required and contains basic information about the created resources and additional options.
+
+### Item parameters
+
+- **id** - _[required]_ - an identifier used to reference the wizard to trigger it's opening.
+- **resources** - _[required]_ - information about the resources created by the wizard. This is a key value map with values consisting of:
+  - **kind** - _[required]_ Kubernetes kind of the resource.
+  - **version** - _[required]_ API version used for all requests.
+  - **group** - API group used for all requests. Not provided for Kubernetes resources in the core (also called legacy) group.
```

Comment:
What about **multiple**

Reply:
// flag for multiple resources of the same kind

Reply:
will be done as follow-up
https://github.com/kyma-project/busola/issues/2431